File: Leetcode/LengthOfLastWord.py
# Given a string s consisting of some words separated by some number of spaces, return the length of the last word in the string.

def lengthOfLastWord(self, s):
    i, length = len(s) - 1, 0
    
    while s[i] == ' ':
        i -= 1
    while s[i] != ' ' and i >= 0:
        length += 1
        i -= 1
    
    return length

File: Leetcode/LengthOfLastWord.cs
// Given a string s consisting of some words separated by some number of spaces, return the length of the last word in the string.

public class Solution 
{
    public int LengthOfLastWord(string s) 
    {
        int i = s.Length - 1, length = 0;
        
        while (s[i] == ' ')
            i--;
        while (i >= 0 && s[i] != ' ')
        {
            length++;
            i--;
        }
        
        return length;
    }
}